### PR TITLE
Add Instagram provider

### PR DESCRIPTION
**Pull request by bocharsky-bw**
In addition to the #7

### CODE REVIEW

File: Client/Provider/InstagramClient.php [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+     * @param AccessToken $accessToken
+     * @return InstagramResourceOwner
+     */
+    public function fetchUserFromToken(AccessToken $accessToken)
+    {
+        return parent::fetchUserFromToken($accessToken);
+    }
+
+    /**
+     * @return InstagramResourceOwner|ResourceOwnerInterface
```
Comment (weaverryan) [inline Client/Provider/InstagramClient.php]: You can remove the `| ResourceOwnerInterface`, since InstagramResourceOwner implements this

**Comment by weaverryan** [comment]
I think this looks good to me :). Is this still WIP?

Btw, I can do it, but if you want, you can execute `php bin/update_readme` to auto-update the README with the documentation for this provider.

Thanks! This is a great provider to have!

**Comment by bocharsky-bw** [comment]
Yes, it's still WIP, I've not tested it yet.

BTW, I've tried to execute `bin/update_readme `, but it doesn't update README for me. I'll look closer on it later, maybe I missed something.

**Comment by weaverryan** [comment]
Ah! That's because I think we forgot one spot: https://github.com/knpuniversity/oauth2-client-bundle/blob/master/DependencyInjection/KnpUOAuth2ClientExtension.php#L31

But yea - once you test it, it will catch any last things we're missing :)

Thanks!

**Comment by bocharsky-bw** [comment]
Done!

**Comment by weaverryan** [comment]
Awesome! Thanks @bocharsky-bw!

**Pull request merged**